src/core/instructions.jl

```diff
@@ -200,10 +200,10 @@ Base.getindex(iter::PhiIncomingSet, i) =
     tuple(Value(API.LLVMGetIncomingValue(ref(iter.phi), Cuint(i-1))),
                 BasicBlock(API.LLVMGetIncomingBlock(ref(iter.phi), Cuint(i-1))))
 
-function Base.append!(iter::PhiIncomingSet, args::Vector{Tuple{Value, BasicBlock}})
+function Base.append!(iter::PhiIncomingSet, args::Vector{Tuple{V, BasicBlock}} where V <: Value)
```

Comment:
If we're going to ditch 0.5 anyway, maybe use the `Tuple{<:Value, BasicBlock}` shorthand?

Reply:
I don't think that works because it would require the Vector to have the same `T` for all `T <: Value`?